### ISSUE TITLE
excludeDefaultStyles documentation is confusing

### ISSUE THREAD

**Issue by DanielaValero** [SC5/sc5-styleguide#1088]
Hello,

To my understanding, the default styles are contained in: styleguide-app. 

If this is correct, and the new option: `excludeDefaultStyles` is set to true, then the `styleguide-app.css` should not be delivered to the app. 

However, at the moment, it is working in the opposite way. If `excludeDefaultStyles` is set to `true`, then the app styles are sent to the app. 

The issue now, is that its default value is `false`, so, when  `excludeDefaultStyles === false`, then the styles are **not** being sent to the app, which causes that the apps of the users now don't get the styles.

To my understanding, we should do any of the next options:

* Rename the option to: `includeDefaultStyles` and set it to `true` by default, to avoid issues in the users
* Fix the behaviour that is implemented at the moment. So: `excludeDefaultStyles === false` send the styles. And `excludeDefaultStyles === true` send the styles. And define it as `false` by default
* Improve the documentation of the new options, so it is clear what it does, and what the default styles are.

**Comment by varya** [comment]
@junaidrsd

**Comment by junaidrsd** [comment]
@DanielaValero styleguide-app.css contains other definitions inside it which is important in both cases for example font styles and custom colors. If excludeDefaultStyle option is set then styleguide does not compile default styles in styleguide-app.css. I’ll make sure both options are working. I think it is better to have false as a default value that is why I set excludeDefaultStyle to true so that user wont effect by this change.

**Comment by DanielaValero** [comment]
Hi @junaidrsd 

I did a new check. I set in my config explicitly`excludeDefaultStyles: false`.

Then the styles are rendered, but not on page load. But somehow, it does not behave in all the cases consistently.

It might be that what happens is that the default value is not being taken properly by the implementation.

However, about the loading consistently or inconsistently, I will try to get more details on the steps to reproduce, and let you know if/when I get more info.
